src/bl/userFacade.ts

```diff
@@ -75,4 +76,20 @@ export class UserFacade {
         }
     }
 
+    // --------------------- Comments ---------------------
+
+    static async addComment(comment: Comment, userId: number): Promise<void> {
+        var user = await UserFacade.getById(userId)
```

Comment:
Maybe I am wrong, but a `Promise<Void>` seems weird. I think returning the User, for what seems to be an update, is not a bad thing.